psoc/grobot_workspace/grobot_module_controller.cydsn/dht11.h

```diff
@@ -0,0 +1,23 @@
+/* ========================================
+ *
+ * Copyright YOUR COMPANY, THE YEAR
+ * All Rights Reserved
+ * UNPUBLISHED, LICENSED SOFTWARE.
+ *
+ * CONFIDENTIAL AND PROPRIETARY INFORMATION
+ * WHICH IS THE PROPERTY OF your company.
+ *
+ * ========================================
+*/
+
+
+#ifndef dht11_h
+#define dht11_h
+    
+#define dhtok 0
+#define dhterror -1
+#define dhttimeout -2
+    
```

Comment:
Wait, you need to put some prototypes here, otherwise I can't use your API!

Also, while we're at it, I think you should use a similar API to most of my other code. In this case, you would have a "dht_init" function that performs global setup at the very beginning. Then, you would have a "dht_start_sample" function that I can call to start taking a new sample. (This would return asynchronously.) After all the interrupts had fired and a sample was ready, I could call "return_decoded_temperature/humidity" and get the most recent sample. (I would actually rename those last functions to "get_decoded_temperature/humidity", again, to be more consistent with my API conventions.)